internal/testing/runchecks.sh

```diff
@@ -174,15 +181,18 @@ echo "Running wire diff..."
 
 echo
 echo "Running Go tests for sub-modules..."
+PATH="$PATH":"${PWD}"/internal/testing
 while read -r path || [[ -n "$path" ]]; do
   echo
   echo "Testing sub-module at '$path'..."
-  echo "  Go tests:"
+  echo "  go test:"
   ( cd "$path" && exec go test -mod=readonly ./... ) || result=1
+  echo "  go mod tidy:"
+  ( cd "$path" && check_mod_tidy.sh && echo "    OK" ) || { echo "FAIL: run go mod tidy on sub-module at '$path'" && result=1; }
   echo "  wire check:"
-  ( cd "$path" && exec wire check ./... && echo "  OK" ) || result=1
```

Comment:
This was broken before; the `echo OK` never ran because of the exec.

Reply:
But why is the "exec" used for "go test" then? Comment on the difference

Reply:
@zombiezen ?

IIUC, the `exec` is just for efficiency, to avoid forking another process.

Reply:
Yes. It's probably safer to just remove them at this point.

Reply:
Done.